core-server/src/test/java/org/orbisgis/orbisserver/ITTests/InContainerIT.java

```diff
@@ -269,7 +269,7 @@ public Result invoke() throws Throwable {
             @Override
             public Result invoke() throws Throwable {
                 return wpsOperationController.displayXML("WPS", "2.0.0", "DescribeProcess",
-                        "file:/C:/Users/mande/AppData/Local/Temp/csvToPointsTable.groovy");
+                        "file:/C:/Users/mande/AppData/Local/Temp/csvToPointsTable.groovy",null);
```

Comment:
Actually, this is the path which is displayed with GetCapabilities, a better idea will be to remove this test.